src/petTrack.js

```diff
@@ -0,0 +1,109 @@
+// PROTOTYPE SYNTAX PET
+
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_FITNESS = 0; 
+const FITNESS_INCREASE = 4;
+const MINIMUM_HUNGER = 0;
+const REDUCE_HUNGER = 3;
+const ACCEPTABLE_HUNGER = 5
+const ACCEPTABLE_FITNESS = 3
+const MAXIMUM_HUNGER = 10
+const MAXIMUM_AGE = 30
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = MINIMUM_HUNGER;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.fitness > MINIMUM_FITNESS && this.hunger < MAXIMUM_HUNGER && this.age < MAXIMUM_AGE;
+    }
+}
+
+Pet.prototype.growUp = function () {
+    
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive');
+      }  
```

Comment:
Couple of things, you dont need a clear line at the beginning of a function definition, and the indentation of the closing bracket is out:
```
    if (!this.isAlive) {
        throw new Error('Your pet is no longer alive');
    }  
```